[docs][fix] Add documentation dependency installation steps

jidicula opened the pull request:
**Why this change was necessary**
The develop installation instructions did not include the
documentation dependencies.

**What this change does**
Adds `docs` to the `pip` installation instructions.

**Any side-effects?**
None

**Additional context/notes/links**

Resolves: #61

po09i commented:
There was a problem when writing the nifti image. The MaRdI property stores the image data as double. The dicomwrite function assumes the data to be between 0 and 1 if the type is double so all values above 1 were changed to the maximum value of uint16. dicomwrite also accepts uint16 and type casting the image as uint16 resolves the issue for uint16 input images. [Fixed in this commit](https://github.com/shimming-toolbox/shimming-toolbox/pull/62/commits/863daceac27a0a0863db663f5c35b8aa37d7eaf0)

However, I assume that not all input images going into the MaRdI.write will have a uint16 types. (phase images for example)

jcohenadad commented:
@po09i wasn't the purpose of this PR to bypass matlab2dcm conversion and to implement direct matlab2nii conversion?

rtopfer commented:
indeed, I would suggest forgetting about `MaRdI.write` entirely: just add a wrapper method to `MaRdI` called `niftiwrite` which calls Matlab's [`niftiwrite`](https://www.mathworks.com/help/images/ref/niftiwrite.html)—most of the arguments (e.g. voxel size, slice orientation etc.) can probably be assigned automatically based on the `MaRdI.Hdr` values.  

i.e. A user with a `MaRdI`-type object `obj` should be able to call `niftiwrite( obj, ... )` and, ideally, be unaware that the call is actually mediated by the `MaRdI.niftiwrite()` wrapper method: the interface should appear identical to the Matlab function itself.

po09i commented:
> wasn't the purpose of this PR to bypass matlab2dcm conversion and to implement direct matlab2nii conversion?

Bypassing dcm2niix was a proposed solution. However, the following problem is originally a problem when writing dicoms.

> The dicomwrite function assumes the data to be between 0 and 1 if the type is double so all values above 1 were changed to the maximum value of uint16.

By fixing this problem, it also fixes the nifti saturation problem when using dcm2niix. 

If we go the niftiwrite wrapper approach, it also means that a dicomwrite wrapper method is required instead of the current write method. I see this solution as cleaner but writing a niftiwrite wrapper is effectively like doing dcm2niix's job since MaRdI is effectively a dicom in a Matlab structure. 

The other solution is writing the dicom properly (which has to be done regardless) and see if dcm2niix satisfies our expectations. I'm happy to go either way.

rtopfer commented:
>If we go the niftiwrite wrapper approach, it also means that a dicomwrite wrapper method is required instead of the current write method.

To be sure, `MaRdI.write` was meant to be precisely that—a wrapper to `dicomwrite`. The nifti functionality got tacked on in a hurry for a specific project, and at a time when Matlab's image processing toolbox didn't have built-in support for nifti (hence dcm2niix).

I think it would be better to get rid of the dcm2niix dependency here and, assuming it works just as well, use niftiwrite instead.

jidicula committed:
[docs][fix] Add documentation dependency installation steps

**Why this change was necessary**
The develop installation instructions did not include the
documentation dependencies.

**What this change does**
Adds `docs` to the `pip` installation instructions.

**Any side-effects?**
None

**Additional context/notes/links**

Resolves: #61

po09i reviewed (approved):
Thanks for the quick fix
